ram/wram.asm

```diff
@@ -399,8 +399,8 @@ NEXTU
 wSimulatedJoypadStatesEnd::
 
 NEXTU
-wBoostExpByExpAll::
-wUnusedCC5B:: db
+wUnusedFlag::
```

Comment:
This byte isn't really unused and it isn't really a flag.

It is all of these labels:

```
00:cc5b wBoostExpByExpAll
00:cc5b wHallOfFame
00:cc5b wFilteredBagItems
00:cc5b wNPCMovementDirections
00:cc5b wDexRatingNumMonsSeen
00:cc5b wVermilionDockTileMapBuffer
00:cc5b wMonPartySpritesSavedOAM
00:cc5b wUnusedFlag
00:cc5b wTrainerCardBlkPacket
00:cc5b wElevatorWarpMaps
00:cc5b wSlotMachineSevenAndBarModeChance
00:cc5b wAnimationType
00:cc5b wOaksAideRewardItemName
```

The two usages of `wUnusedFlag` in the cable club and titlescreen could be trying to clear out any or all of this state. (Maybe the cable club is disabling the Exp All boost.)

I'm just thinking out loud because I don't have any real recommendation.

Reply:
I know that byte has many uses, but those remaining two writes of it are unnecessary. Obviously Game Freak just had this whole UNION as `work_ram: ds 180` or something, so a couple extra clears wouldn't have looked odd.

pokecrystal likewise has `wUnusedPokedexByte` and `wUnusedPokegearByte`, which are inside a UNION of three generic many-use bytes, but in the dex+gear code those two happen to be unnecessary.

Reply:
Yeah, not a big deal, just a comment.